cms/templates/cms/home_page.html

```diff
@@ -91,24 +88,24 @@
              <img src="{% static 'images/online.png' %}" alt="">
           </div>
 
-          <h5 class="upper-case">Online Learning</h5>
-          <p>Take MIT graduate-level courses remotely, from anywhere in the world.</p>
+          <h5 class="upper-case">Convenient & Flexible</h5>
```

Comment:
I believe that ampersands need to be escaped in HTML, so this should be `&amp;` instead of `&`